sdk/src/main/java/io/dapr/actors/Constants.java

```diff
@@ -8,7 +8,7 @@
 /**
  * Useful constants for the Dapr's Actor SDK.
  */
-final class Constants {
+public final class Constants {

```

Comment:
I wonder if we should get rid of this class for Java and keep the constants in the used files, unless we have shared ones. @amanbha  ?

Reply:
?

Reply:
Yes, they can b moved to respective files.